packages/design-system-react/src/components/ButtonHero/ButtonHero.figma.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ButtonHero } from './ButtonHero';
+import figma from '@figma/code-connect';
+
+figma.connect(
+  ButtonHero,
+  'https://www.figma.com/design/1D6tnzXqWgnUC3spaAOELN/%F0%9F%A6%8A-WIP--MMDS-Components?node-id=5416%3A15232',
+  {
+    props: {
+      isDisabled: figma.boolean('isDisabled'),
+      isLoading: figma.boolean('isLoading'),
+    },
+    example: (props) => (
+      <ButtonHero isLoading={props.isLoading} isDisabled={props.isDisabled}>
+        Action
+      </ButtonHero>
+    ),
+  },
+);
```

Comment:
Adding Figma Code Connect to allow for code generation in Figma

https://github.com/user-attachments/assets/bfa7d3ff-4fc2-4cef-84c9-82bd440162e3